test/parallel/test-repl-top-level-await.js

```diff
@@ -84,61 +84,84 @@ async function ordinaryTests() {
   const testCases = [
     [ 'await Promise.resolve(0)', '0' ],
     [ '{ a: await Promise.resolve(1) }', '{ a: 1 }' ],
-    [ '_', '{ a: 1 }' ],
-    [ 'let { a, b } = await Promise.resolve({ a: 1, b: 2 }), f = 5;',
+    [ '_', '// { a: 1 }\r', { line: 0 } ],
+    [ 'let { aa, bb } = await Promise.resolve({ aa: 1, bb: 2 }), f = 5;',
       'undefined' ],
-    [ 'a', '1' ],
-    [ 'b', '2' ],
-    [ 'f', '5' ],
-    [ 'let c = await Promise.resolve(2)', 'undefined' ],
-    [ 'c', '2' ],
-    [ 'let d;', 'undefined' ],
-    [ 'd', 'undefined' ],
-    [ 'let [i, { abc: { k } }] = [0, { abc: { k: 1 } }];', 'undefined' ],
-    [ 'i', '0' ],
-    [ 'k', '1' ],
-    [ 'var l = await Promise.resolve(2);', 'undefined' ],
-    [ 'l', '2' ],
-    [ 'foo(await koo())', '4' ],
-    [ '_', '4' ],
-    [ 'const m = foo(await koo());', 'undefined' ],
-    [ 'm', '4' ],
-    [ 'const n = foo(await\nkoo());', 'undefined' ],
-    [ 'n', '4' ],
+    [ 'aa', ['// 1\r', '1'] ],
+    [ 'bb', ['// 2\r', '2'] ],
+    [ 'f', ['// 5\r', '5'] ],
+    [ 'let cc = await Promise.resolve(2)', 'undefined' ],
+    [ 'cc', ['// 2\r', '2'] ],
+    [ 'let dd;', 'undefined' ],
+    [ 'dd', 'undefined' ],
+    [ 'let [ii, { abc: { kk } }] = [0, { abc: { kk: 1 } }];', 'undefined' ],
+    [ 'ii', ['// 0\r', '0'] ],
+    [ 'kk', ['// 1\r', '1'] ],
+    [ 'var ll = await Promise.resolve(2);', 'undefined' ],
+    [ 'll', ['// 2\r', '2'] ],
+    [ 'foo(await koo())',
+      [ 'f', '// 5oo', '// [Function: foo](await koo())\r', '4' ] ],
```

Comment:
For reviewers: this output looks strange but it's correct. As soon as `f` is entered, the preview is generated and printed. Right afterwards `oo` is entered and the preview for the next variable is printed.